### PR TITLE
Remove deprecated command from GH Actions (DEV-170)

### PR DESCRIPTION
**Pull request by scflode**

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (scflode) [inline .classpath]: meh... need to remove this.

File: .github/workflows/build-and-publish.yml [hunk 2]
```diff
@@ -15,94 +15,96 @@ jobs:
...
+            ./version.txt
+            ./src/main/docker/Dockerfile.native
 
   publish:
     runs-on: ubuntu-latest
     name: Dockerize and publish
     needs: build
+    env:
+      IMAGE_REPO: message-service
+      AWS_REGION: eu-central-1
```
Comment (scflode) [inline .github/workflows/build-and-publish.yml:67]: Static env vars

File: .github/workflows/build-and-publish.yml [hunk 3]
```diff
@@ -15,94 +15,96 @@ jobs:
...
-          ./full_image_name.txt
\ No newline at end of file
+      - name: Fetch artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: message-service
+
+      - name: Set env
+        run: |
+          echo "VERSION=$(cat ./version.txt)" >> ${GITHUB_ENV}
```
Comment (scflode) [inline .github/workflows/build-and-publish.yml:77]: Dynamic env vars

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -12,49 +12,51 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    name:  Deploy to Amazon ECS
+    name: Deploy to Amazon ECS
+    env:
+      AWS_REGION: eu-central-1
+      CLUSTER_NAME: dealog-dev
+      SERVICE_NAME: message-service
+      CONTAINER_NAME: message-service
```
Comment (scflode) [inline .github/workflows/deploy.yml:20]: Static

File: .github/workflows/deploy.yml [hunk 5]
```diff
@@ -12,49 +12,51 @@ on:
...
\ No newline at end of file
+      - name: Download build artifacts
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: ${{ github.event.workflow_run.workflow_id }}
+
+      - name: Set env
+        run: |
+          echo "FULL_IMAGE_NAME=$(cat ./dockerized-message-service/full_image_name.txt)" >> ${GITHUB_ENV}
+          echo "IMAGE_TAG=$(cat ./message-service/version.txt)" >> ${GITHUB_ENV}
```
Comment (scflode) [inline .github/workflows/deploy.yml:31]: Dynamic

File: .github/workflows/deploy.yml [hunk 6]
```diff
@@ -12,49 +12,51 @@ on:
...
+
+      - name: Set production env
+        if: github.ref == 'refs/heads/main'
+        run: echo "CLUSTER_NAME=dealog"
```
Comment (scflode) [inline .github/workflows/deploy.yml]:
```suggestion
        run: echo "CLUSTER_NAME=dealog" >> ${GITHUB_ENV}
```

**Review by ctietze — APPROVED** [review]

**Pull request merged**